epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -252,23 +254,33 @@ private ClassName getGeneratedClassName(TypeElement classElement) {
    * Check each model for super classes that also have attributes. For each super class with
    * attributes we add those attributes to the attributes of the generated class, so that a
    * generated class contains all the attributes of its super classes combined.
+   * <p>
+   * One caveat is that if a sub class is in a different package than its super class we can't
+   * include attributes that are package private, otherwise the generated class won't compile.
    */
   private void updateClassesForInheritance(
```

Comment:
This original method was already dense and hard to read, adding the package stuff on top of it made it denser, so I cleaned it up a little bit.